src/wp-includes/fonts.php

```diff
@@ -92,22 +92,20 @@ function wp_unregister_font_collection( string $slug ) {
 }
 
 /**
- * Returns an array containing the current fonts upload directory's path and URL.
+ * Returns the default font directory path and URL.
  *
  * @since 6.5.0
  *
- * @return array $defaults {
- *     Array of information about the upload directory.
- *
- *     @type string       $path    Base directory and subdirectory or full path to the fonts upload directory.
- *     @type string       $url     Base URL and subdirectory or absolute URL to the fonts upload directory.
- *     @type string       $subdir  Subdirectory
- *     @type string       $basedir Path without subdir.
- *     @type string       $baseurl URL path without subdir.
- *     @type string|false $error   False or error message.
- * }
+ * @param bool $refresh_cache Optional. Whether to refresh the cache and generate a new default font directory path and URL. Default is false.
+ * @return array Default font directory path and URL.
  */
-function wp_get_font_dir() {
+function wp_default_font_dir( $refresh_cache = false ) {
+	$defaults = get_option( 'font_dir' );
```

Comment:
I'm not 100% sure if there are significant benefits in making this persistent. Related: https://github.com/WordPress/gutenberg/issues/59699#issuecomment-1988524392

Reply:
An non-persisted alternative PR to this one: https://github.com/WordPress/wordpress-develop/pull/6259

Reply:
Rather than store the entire font directory in the option, I'd suggest storing whether it falls back to the secondary location.

This will prevent future contributors from having to manage three scenarios:

* default
* fallback
* option set

It's already complicated enough without allowing extenders to change the path via either an option or a filter.